roofit/roofitcore/src/InitUtils.cxx

```diff
@@ -62,35 +30,67 @@ void loadComputeLibrary() {
   }
 
 #if __GNUC__ > 5 || defined(__clang__)
-  if (__builtin_cpu_supports("avx512f")) {
-    libSuffix = "_AVX512f";
+  //skylake-avx512 support
+  if (__builtin_cpu_supports("avx512cd") && __builtin_cpu_supports("avx512vl") && __builtin_cpu_supports("avx512bw") && __builtin_cpu_supports("avx512dq"))  {
+    libSuffix = "_AVX512";
   }
 #endif
 
-#endif
+  if (gEnv->GetValue("RooFit.LoadOptimisedComputationLibrary", 1) == 0) {
+    libSuffix = "";
+    if (gDebug>0) {
+      std::cout << "In roofitcore/Initialisation.cxx:loadComputeLibrary(): RooFit.LoadOptimisedComputationLibrary is set to 0, using generic RooFitCompute library." << std::endl;
+    }
+  }
```

Comment:
Why isn't this the first part of the function, and why doesn't it `return` immediately?
Everything this function is doing is not needed if this is 0.

Reply:
I changed the logic so that the compute library is not part of RooFit Standard package anymore. It needs to load the default if the cpu doesn't support any special instruction sets.